.tekton/discovery-server-pull-request.yaml

```diff
@@ -255,20 +234,16 @@ spec:
         - $(params.build-args[*])
       - name: BUILD_ARGS_FILE
         value: $(params.build-args-file)
+      - name: PRIVILEGED_NESTED
+        value: $(params.privileged-nested)
       - name: SOURCE_ARTIFACT
         value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
       - name: CACHI2_ARTIFACT
         value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
       - name: IMAGE_APPEND_PLATFORM
         value: "true"
-      # Customization added to allow using the version extracted from package.json
-      # as version label
-      - name: LABELS
-        value:
-          - $(tasks.generate-version-labels.results.labels[*])
       runAfter:
       - prefetch-dependencies
-      - generate-version-labels
       taskRef:
```

Comment:
the custom task for generating labels and the labels above are required per our RPA. are we supposed to change that too?